Modules/Package.swift

```diff
@@ -312,7 +312,6 @@ enum XcodeSupport {
             .xcodeTarget("XcodeTarget_App", dependencies: keystoneDependencies),
             .xcodeTarget("XcodeTarget_Keystone", dependencies: keystoneDependencies),
             .xcodeTarget("XcodeTarget_WordPressTests", dependencies: testDependencies + [
-                "SFHFKeychainUtils",
```

Comment:
The remaining one is [below at line 324](https://github.com/wordpress-mobile/WordPress-iOS/pull/24514/files#diff-0386b743aa88904e7553ee4d8aa3da26d736bf86ea6d1a6d6b78bf541181035cR324)